xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/internal/skin/AbstractResourceSkin.java

```diff
@@ -132,6 +136,14 @@ protected Configuration getProperties()
 
                 this.properties = new BaseConfiguration();
             }
+        } else {
+            try {
+                Path resourcePath = Paths.get(getResourceURL(getPropertiesPath()).toURI());
+                FileTime lastModifiedTime = Files.getLastModifiedTime(resourcePath);
+            } catch (Exception e) {
```

Comment:
Not even a log or something?

Reply:
Looks like I forgot (otherwise I would have put a comment).

Reply:
Actually that code has no reason to exist at all (and it does nothing). I must have forgot to remove it.